common/src/main/resources/data/conditional/recipes/farm_and_charm/mincer/cornmeal_from_ce.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "doapi:conditional",
+  "recipes": [
+    {
+      "conditions": [
+        {
+          "type": "doapi:not",
+          "value": {
+            "type": "forge:mod_loaded",
+            "modid": "brewery"
+          }
+        }
+      ],
+      "recipe": {
+        "type": "farm_and_charm:mincer",
+        "ingredient": {
+          "item": "cornexpansion:dried_corn"
+        },
+        "recipe_type": "MEAT",
```

Comment:
Should the recipe_type be "MEAT" ?

Reply:
Similar to the silo every example I saw for the mincer shows as MEAT type. I'll take a look and see if there are other types that are in use that'd fit better